### ISSUE TITLE
[Chromaticraft] Lumen Beacon interfering with itself

### ISSUE THREAD

**Issue by Solarshade** [ReikaKalseki/Reika_Mods_Issues#3405]
Whenever the Lumen Beacon in my singleplayer world is unloaded and reloaded it ceases to work. With some investigation, it was concluded that it is seemingly due to the interference mechanic of beacons being triggered by itself:

Beacon pre-interference:
![Pre-Intereference](https://github.com/user-attachments/assets/ccb8b5e6-e27b-48a9-84cf-a9ce1a57ee54)

Beacon Post unloading and reloading:
![Post-Interference](https://github.com/user-attachments/assets/49a2b800-9868-4005-a091-0582b0f99506)

This seemingly occurs regardless of whether or not the chunk the beacon is in is being loaded or not, or at the very least is not affected by Railcraft Anchors.

**Comment by MalTeeez** [comment]
Does your multi flag also change across these 2 states?

**Comment by Solarshade** [comment]
I'm unsure what you mean by this? Multi-Repeaters are unaffected by this though, they remain functional. If you mean the multi-repeater at the bottom of the Beacon, it doesn't seem to change. It doesn't have the interference mechanic.

**Comment by MalTeeez** [comment]
have a look at using DragonAPI's debug key for seeing tile data, I think your mod is excluding some tags.

**Comment by MalTeeez** [comment]
Nevermind, my bad, seems like that changed a few versions ago. 
Although after building one and testing a little, I don't have this issue (with a dedicated server, not Singleplayer) - Have you tried putting your world onto a separate server?

**Comment by Solarshade** [comment]
Yes, I'm aware that this is solved by switching to a dedicated server, that's why I specified it was in a singleplayer world. Speculation from people I talked about is that this is likely because the beacon tile-entity is somehow being temporarily duplicated, and as the check to prevent self-interference only checks if it's the same tile-entity, this duplication triggers it. Likely could be fixed by having beacons check if the interfering beacon has the same coordinates.
